gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 
 # ArcGIS Maps SDK for Kotlin version
-arcgisMapsKotlinVersion = "200.7.0-4516"
+arcgisMapsKotlinVersion = "200.7.0-4541"
```

Comment:
I needed to update the existing v.Next version in order to pick up final API changes.